executing/executing.py

```diff
@@ -581,11 +541,12 @@ def __init__(self, frame, stmts, tree, lasti, source):
         elif op_name in ('LOAD_ATTR', 'LOAD_METHOD', 'LOOKUP_METHOD'):
             typ = ast.Attribute
             ctx = ast.Load
-            extra_filter = lambda e: attr_names_match(e.attr, instruction.argval)
+            extra_filter = lambda e:mangled_name(e) == instruction.argval 
         elif op_name in ('LOAD_NAME', 'LOAD_GLOBAL', 'LOAD_FAST', 'LOAD_DEREF', 'LOAD_CLASSDEREF'):
             typ = ast.Name
             ctx = ast.Load
-            extra_filter = lambda e: e.id == instruction.argval
+            if instruction.argval:
```

Comment:
The original was `if sys.version_info[0] == 3 or instruction.argval:`, with an **or**, so is this check needed?

Reply:
it is not needed 4 lines above and it is most likely not needed here.